design_temp/clean.py

```diff
@@ -0,0 +1,147 @@
+import csv
+import re
+from dataclasses import dataclass
+
+
+# This script generates design_temp_by_county.csv
+# from the raw text copied from the ENERGY STAR pdf located at:
+# https://www.energystar.gov/ia/partners/bldrs_lenders_raters/downloads/County%20Level%20Design%20Temperature%20Reference%20Guide%20-%202015-06-24.pdf
+
+
+def remove_junk(content):
+    table_title = (
+        "Exhibit 1: Cooling and Heating Design Temperatures by State and County"
+    )
+    content = content.split(table_title)[1]
+    pattern = r"[ ]+ENERGY STAR Certified Homes\nCounty-Level Design Temperature Reference Guide"
+    content, subs = re.subn(pattern, "", content)
+    if subs != 115:
```

Comment:
I think I can follow the regex replacement happening here, but perhaps add some comment (either inline or add the start of the block) on what exactly is being trimmed in these steps.